adjust

```diff
@@ -632,7 +634,7 @@ def wait_for_update(appname, obj, patch_gen, print_progress, c=0, t=1, wait_for_
 def set_rsrc(cp, sn, sv, sel='both'):
     rn = RESOURCE_MAP[sn]
     if sn == "mem":
-        sv = str(round(sv,3)) + 'Gi'     # internal memory representation is in GiB
+        sv = '{}Gi'.format(float(round(sv * Gi)) / Gi) # Round to nearest byte, internal memory representation is in GiB
```

Comment:
What problem is this solving? Are you aware of a value that is handled incorrectly by the previous version and how is it handled with this change?

Reply:
I've updated the rounding of memory per this task: https://app.asana.com/0/1152347948866655/1175879790190716

> Also, ensure that the final resource values that will be given to k8s to set are rounded to the nearest [1 byte for memory]

Reply:
Are you aware of a value that is handled incorrectly by the previous version and how is it handled with this change?

Reply:
No not personally, I am just following the spec from the task. 

@pnickolov are you aware of any cases where the memory rounding has been handled incorrectly by the current version of servo-k8s?

Reply:
What I am asking is - the previous version already does rounding, how is the new one different? If I read it correctly, given an input value of 0.0125, the old version would round to `0.013Gi` while the new version would produce `0.012500000186264515Gi`. Is that the desired behaviour?